utils/cosmos.ts

```diff
@@ -18,19 +17,60 @@ export const royaltyRateBasisPoints = 1000 // 10% as in current chain config
 export const royaltyFeeAmount = 25000 // 2.5%
 export const royaltyUserAmount = 1000000 - royaltyFeeAmount // 1000000 - fee
 
-let iscnSigningClient: ISCNSigningClient | null = null
+let iscnSigningClient: any = null
+let iscnLib: any = null
+let iscnParsingLib: any = null
+let cosmLib: any = null
+let iscnMsgLib: any = null
+let likenftMsgLib: any = null
+
+async function getISCNLib () {
+  if (!iscnLib) {
+    iscnLib = await import(/* webpackChunkName: "iscn" */ '@likecoin/iscn-js')
+  }
+  return iscnLib
+}
+
+async function getISCNParsingLib () {
+  if (!iscnParsingLib) {
+    iscnParsingLib = await import(/* webpackChunkName: "iscn-parsing" */ '@likecoin/iscn-js/dist/iscn/parsing')
+  }
+  return iscnParsingLib
+}
 
-export async function getSigningClient (): Promise<ISCNSigningClient> {
+async function getISCNMsgLib () {
+  if (!iscnMsgLib) {
+    iscnMsgLib = await import(/* webpackChunkName: "iscn-msg" */ '@likecoin/iscn-js/dist/messages/parsing')
+  }
+  return iscnMsgLib
+}
+
+async function getLikeNFTMsgLib () {
+  if (!likenftMsgLib) {
+    likenftMsgLib = await import(/* webpackChunkName: "likenft-msg" */ '@likecoin/iscn-js/dist/messages/likenft')
+  }
+  return likenftMsgLib
+}
```

Comment:
I think we can merge all these into `getISCNLib`, then parse the individual functions out from the main lib

Reply:
[♻️ Refactor ISCN library imports to consolidate and simplify usage](https://github.com/likecoin/nft-book-press/pull/320/commits/ad11d3b863a48cea1c1ee76b3766ab507ad1a4e2)